tokio/src/loom/std/atomic_ptr.rs

```diff
@@ -1,7 +1,6 @@
 use std::fmt;
 use std::ops::{Deref, DerefMut};
 
-/// `AtomicPtr` providing an additional `load_unsync` function.
```

Comment:
This no longer provides any extra functions - `loom` directly imports `std::sync::atomic::AtomicBool` because it doesn't need to add extras.

Would it make sense to remove this file completely, and change `tokio/src/loom/std/mod.rs` to use `std::sync::atomic::AtomicPtr` directly?